### DISCUSSION TITLE
How to get cookie if cookie name does have dash in it like d-zz-a

### DISCUSSION THREAD

**Discussion by ArunKumarSwain** [fastapi/fastapi#6493] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
FIle Name: test.py
--------------------------------------------------------------------------------------------------------
from fastapi import FastAPI, Cookie
import uvicorn
from typing import Optional

app = FastAPI()

@app.get("/")
def getCookieValue(d_zz_a:Optional[str]=Cookie(None)):
    return {"cookie":d_zz_a}
    
if __name__ == "__main__":
    uvicorn.run("test:app", host="127.0.0.1", port=8000, log_level="info")
----------------------------------------------------------------------------------------------------

curl to call:
curl --location --request GET 'http://127.0.0.1:8000/' \
--header 'Cookie: d-zz-a=cookievalue'
```

### Description

My requirement is to get the cookie value of 'd-zz-a', but I can't define d-zz-a as parameter variable in getCookieValue function like

Python does not allow this:
def getCookieValue(d-zz-a:Optional[str]=Cookie(None)):
    return {"cookie":d-zz-a}

What is the way to get the d-zz-a cookie value?

### Operating System

Windows

### FastAPI Version

0.70.1

### Python Version

3.9.2

### Additional Context

![image](https://user-images.githubusercontent.com/10923121/149610355-d35a5ea4-c05e-40ab-906b-62e4e0186fad.png)

**Comment by n8sty** [comment, marked as answer]
`Cookie` should have an `alias` parameter that you can use for this

**Comment by ArunKumarSwain** [comment]
Cookie alias worked for me:
---------------------------------------------------------------------------------------------------------------------------------------
from fastapi import FastAPI, Cookie, Request
import uvicorn
from typing import Optional

app = FastAPI()

@app.get("/")
def getDeviceProperties(d_zz_a:Optional[str]=Cookie(default=None,alias='d-zz-a'), requests=Request):
    return {"cookie":d_zz_a}
    
if __name__ == "__main__":
    uvicorn.run("test:app", host="127.0.0.1", port=8000, log_level="info")
